drivers/network/afd/afd/lock.c

```diff
@@ -69,7 +69,10 @@ PVOID LockRequest( PIRP Irp,
                 }
 
                 /* The allocated address goes in index 0 */
-                Irp->Tail.Overlay.DriverContext[0] = ExAllocatePool(NonPagedPool, MmGetMdlByteCount(Irp->MdlAddress));
+                Irp->Tail.Overlay.DriverContext[0] = ExAllocatePoolWithTag(NonPagedPool,
```

Comment:
This (and AfdInfo below) could have its own tag; there's not really a connection to Recv.Window/Send.Window, right?

Reply:
Sure, what do you have in mind?